python/netifaces/defs.py

```diff
@@ -1,7 +1,12 @@
 from enum import IntEnum
 from typing import Dict, List, Tuple, Union
 
-from typing_extensions import Literal
+try:
+    # python >=3.8
+    from typing import Literal
+except ImportError:
+    # python <3.8
+    from typing_extensions import Literal
```

Comment:
This would be the correct way to write this so `mypy` does not whine:
```diff
-try:
-    # python >=3.8
-    from typing import Literal
-except ImportError:
-    # python <3.8
+import sys
+
+if sys.version_info < (3, 8):
     from typing_extensions import Literal
+else:
+    from typing import Literal
```